### PR TITLE
fix(docs): mention to add your own icon inside a directory

### PR DESCRIPTION
**Pull request by hemakshis**
Issue #8366 - To inform the user regarding the need to supply an icon in part 8 of the tutorial as well as in add a manifest file page of the documentation.

Addressing the issues mentioned in #9581

### CODE REVIEW

**Comment by hemakshis** [comment]
@wardpeet I have made the requested changes. Please let me know if this looks good to you :grinning:

**Comment by calcsam** [comment]
@shannonbux @marcysutton could you comment about whether this change is desired?

**Comment by shannonbux** [comment]
Hi @calcsam, according to the issue @amberleyromo created (#8366), which is mentioned at the top of this PR comment thread, it is. As far as wording goes, @amberleyromo and @marcysutton would be the ones to ask @wardpeet! Thanks!

File: docs/tutorial/part-eight/index.md [hunk 1]
```diff
@@ -85,7 +85,9 @@ Quoting [Google](https://developers.google.com/web/fundamentals/web-app-manifest
 npm install --save gatsby-plugin-manifest
 ```
 
-2.  Add the plugin to the `plugins` array in your `gatsby-config.js` file.
+2. Add a favicon for your app under `src/images/icon.png`. An icon option is needed to not break the [`manifest.json`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-manifest/README.md) file.
```
[the remark this answers is not in the document]
Comment (marcysutton) [reply docs/tutorial/part-eight/index.md]: These suggestions look good to me. Once they are made, we can merge!

**Comment by hemakshis** [comment]
Hi @marcysutton, I have made the suggested changes and the PR is ready to merge.

**Pull request merged**